Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -782,10 +779,15 @@ final class SettingsTableViewController: UITableViewController, IdentifiableClas
                                     bolusVolumes: $0.supportedBolusVolumes,
                                     maximumBasalScheduleEntryCount: $0.maximumBasalScheduleEntryCount)
         }
+        let servicesViewModel = ServicesViewModel(showServices: !Bundle.main.bundleDisplayName.contains("Tidepool"), // Hack for now, until we get rid of this VC
```

Comment:
Sorry for reverting the previous approval. This should be changed to a FeatureFlag. This will continue to be used in DIY Loop, but might be turned off in Tidepool Loop

Reply:
Yeah I was wondering about this... the "hack" I put in is not really great.  I'll add a FeatureFlag for Services.